packages/next/src/shared/lib/fnv1a.ts

```diff
@@ -1,6 +1,7 @@
 // source: https://github.com/sindresorhus/fnv1a
 // FNV_PRIMES and FNV_OFFSETS from
 // http://www.isthe.com/chongo/tech/comp/fnv/index.html#FNV-param
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
```

Comment:
Because this is copy & pasted from an external source that deals with numbers, I don't think we really needed to flag for correct behavior around numeric precision. 😄 

For reference, https://typescript-eslint.io/rules/no-loss-of-precision/ (and also https://github.com/typescript-eslint/typescript-eslint/issues/7277 noting that the typescript-eslint extension might not be necessary anymore)